webpack/assets/javascripts/react_app/redux/index.js

```diff
@@ -6,7 +6,11 @@ import reducers from './reducers';
 
 let middleware = [thunk];
 
-if (process.env.NODE_ENV !== 'production' && !global.__testing__) {
+if (
+  process.env.NODE_ENV !== 'production' &&
+  process.env.REDUX_LOGGER !== false &&
```

Comment:
I wonder if we should keep the old behavior (e.g. set it to false if you are in production) but allow to override for production too (e.g. if you want to debug a production deployment)

Reply:
This is meant for developers (like me :) ) who don't want the logger as it bloats the console

Reply:
I understand, now if I ask you to debug a production env instead?

Reply:
added `else` condition for enabling the logger in production if needed